src/samplers/scheduler/mod.rs

```diff
@@ -306,3 +326,37 @@ impl Scheduler {
         Ok(())
     }
 }
+
+#[cfg(feature = "bpf")]
+fn to_map(table: &bcc::table::Table) -> std::collections::HashMap<u32, u64> {
```

Comment:
Does it make sense to have this fn in common?

Reply:
Fixed